Google.PowerShell.IntegrationTests/BigQuery/GcbqCmdlets.ps1

```diff
@@ -0,0 +1,16 @@
+﻿. $PSScriptRoot\..\GcloudCmdlets.ps1
```

Comment:
I can take out the Test-Funct, since that was just a reminder to myself about syntax.  As for the file, I saw that pattern being used to define constants and functions in between the different sets of cmdlet tests in other sections.  It should start to become useful for future sets, such as tables and datasets.  Is it better to leave it until then or take it out now and include it later when it's explicitly needed?